src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -491,6 +497,9 @@ private String getTaskLoggingTag() {
    */
   @VisibleForTesting
   static KafkaRecordErrorReporter noOpKafkaRecordErrorReporter() {
-    return (record, e) -> {};
+    return (record, e) -> {
+      STATIC_LOGGER.warn(
```

Comment:
same here, is it easy to add a test?

Reply:
Added a test, a lot of plumbing but I would say worth it. We now have unit test, IT and E2E test coverage.

Reply:
Yes, this is essentially a no-op. E2E tests covers this.
Added a coverage through an in memory implementation of the error reporter (which can be thought of as a DLQ)